dwango/fialyzer
# Add blackbox tests

I have added templates for blackbox testing. resolves #38 

- `make test` requires Erlang/OTP now.
  - `make unit-test` is the same behavior of previous `make test`.
- In CI, Erlang files are compiled and used for blackbox test by CircleCI workflow.

### How to add blackbox test

1. Add `test/blackbox-test/test-cases/foo.erl`
1. Add `test/blackbox-test/test-cases/foo.expected`, this is the expected output of `fialyzer foo.beam`
1. Run `make test` or `make beam && make blackbox-test`
